src/Form/QuestionType/QuestionTypeDateTime.php

```diff
@@ -0,0 +1,374 @@
+            'is_default_value_current_time' => $this->isDefaultValueCurrentTime($question),
+            'placeholders'      => $this->getPlaceholders()
+        ]);
+    }
+
+    #[Override]
+    public function renderAdministrationOptionsTemplate(?Question $question): string
+    {
+        $template = <<<TWIG
+            {% set rand = random() %}
+
+            <div class="d-flex gap-2">
+                <label class="form-check form-switch mb-0">
+                    <input type="hidden" name="is_date_enabled" value="0"
+                    data-glpi-form-editor-specific-question-extra-data>
+                    <input class="form-check-input" type="checkbox" name="is_date_enabled"
+                        id="is_date_enabled_{{ rand }}"
+                        value="1" {{ is_date_enabled ? 'checked' : '' }}
+                        onchange="handleDateAndTimeCheckbox_{{ rand }}(this)"
+                        data-glpi-form-editor-specific-question-extra-data>
+                    <span class="form-check-label">{{ labels.date|e('html') }}</span>
+                </label>
+                <label class="form-check form-switch mb-0">
+                    <input type="hidden" name="is_time_enabled" value="0"
+                    data-glpi-form-editor-specific-question-extra-data>
+                    <input class="form-check-input" type="checkbox" name="is_time_enabled"
+                        id="is_time_enabled_{{ rand }}"
+                        value="1" {{ is_time_enabled ? 'checked' : '' }}
+                        onchange="handleDateAndTimeCheckbox_{{ rand }}(this)"
+                        data-glpi-form-editor-specific-question-extra-data>
+                    <span class="form-check-label">{{ labels.time|e('html') }}</span>
+                </label>
+            </div>
+
+            <script>
+                {# Both date and time can be checked at the same time, but one of them must be checked #}
+                function handleDateAndTimeCheckbox_{{ rand }}(input) {
+                    const isChecked = $(input).is(':checked');
+                    const otherInput = $('input[onchange^="handleDateAndTimeCheckbox_{{ rand }}"]:not([name="' + input.name + '"])');
+
+                    if (!isChecked && otherInput.not(':checked')) {
+                        otherInput.prop('checked', true);
+                    }
+
+                    updateDateAndTimeInputType($(input).closest('div[data-glpi-form-editor-question]'));
+                }
+            </script>
+TWIG;
+
+        $twig = TemplateRenderer::getInstance();
+        return $twig->renderFromStringTemplate($template, [
+            'question' => $question,
+            'is_date_enabled' => $this->isDateEnabled($question),
+            'is_time_enabled' => $this->isTimeEnabled($question),
+            'labels' => [
+                'date' => _sn('Date', 'Dates', 1),
+                'time' => _sn('Time', 'Times', 1)
```

Comment:
Will be automatically escaped by Twig.
```suggestion
                'date' => _s('Date', 'Dates', 1),
                'time' => _s('Time', 'Times', 1)
```